pom.xml

```diff
@@ -374,6 +374,10 @@
             <groupId>commons-logging</groupId>
             <artifactId>*</artifactId>
           </exclusion>
+          <exclusion>
```

Comment:
PR to fix this upstream: https://github.com/NationalSecurityAgency/datawave/pull/2552